source/fundamentals/stable-api.txt

```diff
@@ -1,6 +1,8 @@
-=============
-Versioned API
-=============
+.. _stable-api:
```

Comment:
Comment: Consider prefixing this `ref` with Node, as this is language specific content (reserve th ``.. _stable-api`` ref for the central page that covers the stable API)

```suggestion
.. _node-stable-api:
```

Reply:
Good catch. Just FYI, there's no directive planned for a central page afaik. The manual reference page that will be published (to replace the old one) is on line 38.